### PR TITLE
v1beta6

### PR DESCRIPTION
**Pull request by balopat**

### CODE REVIEW

File: pkg/skaffold/schema/v1beta5/upgrade.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+		APIVersion: next.Version,
+		Kind:       config.Kind,
+		Build:      newBuild,
+		Test:       newTest,
+		Deploy:     newDeploy,
+		Profiles:   newProfiles,
+	}, nil
+}
+
+func convert(old interface{}, new interface{}) error {
```
Comment (priyawadhwa) [inline pkg/skaffold/schema/v1beta5/upgrade.go]: This is the same as the function [CloneThroughJSON](https://github.com/GoogleContainerTools/skaffold/blob/master/pkg/skaffold/util/util.go#L260)  I added to the util package for build plugins, maybe we should use that instead?

**Comment by dgageot** [comment]
LGTM but needs to be rebased

File: docs/content/en/schemas/v1beta5.json [hunk 2]
```diff
@@ -326,7 +326,7 @@
         },
         "image": {
           "type": "string",
-          "description": "Docker image used by the Kaniko pod. Defaults to the latest released version of <code>gcr.io/kaniko-project/executor</code>."
+          "description": "Docker image used by the Kaniko pod. Defaults to the v1beta5 released version of <code>gcr.io/kaniko-project/executor</code>."
```
Comment (dgageot) [inline docs/content/en/schemas/v1beta5.json]: @balopat this is not supposed to be replaced :-)

Comment (balopat) [reply docs/content/en/schemas/v1beta5.json]: LOL nice catch!!!!

File: pkg/skaffold/schema/v1beta5/config.go [hunk 3]
```diff
@@ -0,0 +1,591 @@
...
+	// Namespace is the Kubernetes namespace.
+	// Defaults to current namespace in Kubernetes configuration.
+	Namespace string `yaml:"namespace,omitempty"`
+
+	// Timeout is the amount of time (in seconds) that this build is allowed to run.
+	// Defaults to 20 minutes (`20m`).
+	Timeout string `yaml:"timeout,omitempty"`
+
+	// Image is the Docker image used by the Kaniko pod.
+	// Defaults to the v1beta5 released version of `gcr.io/kaniko-project/executor`.
```
[the remark this answers is not in the document]
Comment (balopat) [reply pkg/skaffold/schema/v1beta5/config.go]: sigh. thanks for catching these!

File: pkg/skaffold/schema/v1beta5/upgrade.go (hunk 1 above)
Comment (balopat) [reply pkg/skaffold/schema/v1beta5/upgrade.go]: good catch, I think that's a good idea. we'll separate them if necessary.

File: pkg/skaffold/schema/v1beta5/config.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (balopat) [reply pkg/skaffold/schema/v1beta5/config.go]: actually, this was the source of the previous one through schema generation. Fixed!

**Pull request merged**